src/modules/rppt_tensor_geometric_augmentations.cpp

```diff
@@ -1017,6 +1017,106 @@ RppStatus rppt_slice_host(RppPtr_t srcPtr,
     return RPP_SUCCESS;
 }
 
+/******************** normalize_ND ********************/
+
+RppStatus rppt_normalize_generic_host(RppPtr_t srcPtr,
```

Comment:
Normalize is in geometric here. Not sure whats the logic. Pls consolidate, if statistical makes more sense for a mean/stdDev based normalization function.

Reply:
Moved to statistic